### ISSUE TITLE
Bash script has syntax error

### ISSUE THREAD

**Issue by kulovan** [whyayen/bye-github-draft#1]
Hi,

I really appreciate the orb you created, we would make good use of it to filter out ineligible PRs.
I found an error in the bash script. In the first line, there is a comma that is unnecessary and it is appended to the `GITHUB_TOKEN`.

```
required_env_vars=(
              "GITHUB_TOKEN",
              "CIRCLE_TOKEN"
            )
          
```
I would happily open a PR to fix this. However, I suspect from the repo structure, that publishing to CircleCI orb hub is not an automated process, so maybe you have to do that manually.

I found another issue, but I will submit it separately.

**Comment by whyayen** [comment]
Sorry for late replying.

I have no idea why i don't get syntax error warning. It works as expected. 😆
I appreciate your help very much. I would review & test PR #3 as soon as possible.

**Comment by kulovan** [comment]
Hi @whyayen,

Sorry, I was ill, I could not reply.

It does not throw a syntax error, it just messes up the `GITHUB_TOKEN` because it appends an extra `,` to it. Also, I think it depends on the shell. Did you try to run the command in `alpine:3.7` docker container ?

**Comment by whyayen** [comment]
> Hi @whyayen,
> 
> Sorry, I was ill, I could not reply.
> 
> It does not throw a syntax error, it just messes up the `GITHUB_TOKEN` because it appends an extra `,` to it. Also, I think it depends on the shell. Did you try to run the command in `alpine:3.7` docker container ?
> 

@kulovan The script works, but Circle CI can't trigger she'll script correctly. I think my Circle CI orb config wrong. I'll fix it on the weekend. Let me try it! Take good care of yourself ❤️

**Comment by whyayen** [comment]
Merged #3 . Thank for your help ! :)
